charts/infra-apps/templates/namespaces.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.namespaces.enabled }}
```

Comment:
As noted, I'm not sure if this makes sense here. Maybe namespaces should get created by argo events/workflow. The event/workflow could also take care of actually setting up `*-apps` helm charts.